lib/expansion_rules.rb

```diff
@@ -18,6 +18,7 @@ def recurring
     [:parent.recurring],
     [:parent_taxons.recurring],
     [:parent_taxons.recurring, :root_taxon],
+    %i[taxons root_taxon],
```

Comment:
To improve readability, I'd prefer to always use the `[:taxons, :root_taxon]` representation.

Reply:
rubocop says different :(

Reply:
Maybe disable that check for this file? I think you can add a comment of this form `# rubocop:disable ...` [1].

1: https://github.com/bbatsov/rubocop/blob/master/manual/configuration.md

Reply:
Good idea! Added that in new commit. 👍